jvcl/run/JvInterpreterParser.pas

```diff
@@ -433,6 +433,7 @@ function TokenTyp(const Token: string): TTokenKind;
   T1: Char;
   Ci: Char;
   Point: Boolean;
+  sci_nota:Boolean;
```

Comment:
The name of this variable does not respect the coding guide. Please rename it to `IsScientificNotation`